gardening/data/product_product.xml

```diff
@@ -0,0 +1,70 @@
+    <field name="product_tmpl_id" ref="product_template_7" />
+    <field name="product_template_attribute_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_2')])]" />
+    <field name="product_template_variant_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_2')])]" />
+  </record>
+  <record id="product_product_10" model="product.product">
+    <field name="product_tmpl_id" ref="product_template_7" />
+    <field name="product_template_attribute_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_3')])]" />
+    <field name="product_template_variant_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_3')])]" />
+  </record>
+  <record id="product_product_11" model="product.product">
+    <field name="product_tmpl_id" ref="product_template_8" />
+    <field name="product_template_attribute_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_4')])]" />
+    <field name="product_template_variant_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_4')])]" />
+  </record>
+  <record id="product_product_12" model="product.product">
+    <field name="product_tmpl_id" ref="product_template_8" />
+    <field name="product_template_attribute_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_5')])]" />
+    <field name="product_template_variant_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_5')])]" />
+  </record>
+  <record id="product_product_13" model="product.product">
+    <field name="product_tmpl_id" ref="product_template_9" />
+    <field name="product_template_attribute_value_ids"
+      eval="[(6, 0, [ref('product_template_attribute_value_6')])]" />
+  </record>
+  <record id="product_product_14" model="product.product">
+    <field name="name">Labor with equipment</field>
+    <field name="type">service</field>
+    <field name="categ_id" ref="product.product_category_all" />
+    <field name="service_policy">delivered_timesheet</field>
+    <field name="route_ids" eval="[(6, 0, [ref('purchase_stock.route_warehouse0_buy')])]" />
+    <field name="service_tracking">task_in_project</field>
+    <field name="list_price">120.0</field>
+    <field name="project_template_id" ref="project_project_10"/>
+  </record>
+  <record id="product_product_15" model="product.product">
+    <field name="name">Labor hours</field>
+    <field name="categ_id" ref="product.product_category_all" />
+    <field name="type">service</field>
+    <field name="service_policy">delivered_timesheet</field>
+    <field name="route_ids" eval="[(6, 0, [ref('purchase_stock.route_warehouse0_buy')])]" />
+    <field name="service_tracking">task_in_project</field>
+    <field name="list_price">60.0</field>
+    <field name="project_template_id" ref="project_project_10" />
+  </record>
+  <record id="industry_fsm_sale.field_service_product" model="product.product">
```

Comment:
I'd rather create a new record

Reply:
Then do I modify the other modules that implements it this way?